05_syntax_lab/05.py

```diff
@@ -3,3 +3,15 @@
 until it finds a number that is divisable by: 7, 13 and 15
 """
 
+from random import randint
+
+numberToDiviseto = 7*13*15
+
+
+while True:
+    num = randint(1,10000000)
+    if num % numberToDiviseto ==0:
```

Comment:
In python we prefer lowercase variable names + spaces around equality operators. So prefer `if num % number_to_diviseto == 0`